core/src/main/java/ai/timefold/solver/core/config/solver/SolverConfig.java

```diff
@@ -280,6 +290,22 @@ public SolverConfig(@NonNull SolverConfig inheritedConfig) {
         inherit(inheritedConfig);
     }
 
+    /**
```

Comment:
We are using Clock for purposes other than testing. In several locations, `System.currentTimeMillis()` has been replaced with `getSolverScope().getClock().millis()`.